### PR TITLE
planner: Fix more cases that explain does not display a truncated value

### PR DESCRIPTION
**Pull request by EricZequan**
### What problem does this PR solve?

Issue Number: ref #54245 

Problem Summary:

### What changed and how does it work?
This PR brings long constant truncation to more kind of plans.

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/util/intest/common.go [hunk 1]
```diff
@@ -17,7 +17,7 @@
 package intest
 
 // InTest checks if the code is running in test.
-const InTest = false
+const InTest = true
```
Comment (hawkingrei) [inline pkg/util/intest/common.go:20]: Please don't change it.

Comment (EricZequan) [reply pkg/util/intest/common.go:20]: modified.

**Comment by breezewish** [comment]
Close for https://github.com/pingcap/tidb/pull/55934

**Pull request closed without merging**